### PR TITLE
Releasing 0.7.3

### PR DESCRIPTION
**Pull request by benlower**
Patch to update the readme on npm.

### CODE REVIEW

File: packages/docs/docs/changelog.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 0.7.2
+## 0.7.3
+
+- Update readme on npm.
+
+## [0.7.2](https://github.com/fixie-ai/ai-jsx/pull/221/commits)
```
Comment (NickHeiner) [inline packages/docs/docs/changelog.md:7]: Please link to the actual commit, which in this case is 203574abdbdce22c876a0c5a3a94dcc093b753cb.

**Review by zkoch — APPROVED** [review]

File: packages/docs/docs/changelog.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 0.7.2
+## 0.7.3
+
+- Update readme on npm.
```
Comment (NickHeiner) [inline packages/docs/docs/changelog.md:5]:
```suggestion
- Update readme.
```

It's not just updated on npm.

**Review by NickHeiner — APPROVED** [review]

File: packages/docs/docs/changelog.md (hunk 2 above)
Comment (benlower) [reply packages/docs/docs/changelog.md:5]: the readme on github was updated separately so this change actually only changes on npm.

File: packages/docs/docs/changelog.md (hunk 1 above)
Comment (benlower) [reply packages/docs/docs/changelog.md]: ok fixed

**Pull request merged**